doc/vim-go.txt

```diff
@@ -693,20 +694,25 @@ CTRL-t
     [dir] is given it parses the given directory.
 
                                                                  *unite-decls*
+                                                                *denite-decls*
 :Unite decls[:file or dir]
+:Denite decls[:file or dir]
```

Comment:
`:Denite decls dir` doesn't seem to work? I get `[denite] NameError: Source "dir" is not found.`.

Reply:
denite needs options `:` separated.  And the `file or dir` is some illustration, so users should supply the real path.

```vim
:Denite decls:~/.go/src/github.com/golang/appengine
" relative path also available. same meaning if you are in ~/.go/src/github.com
:Denite decls:golang/appengine
```

The description is a bit obscure maybe, but I followed descriptions already written about unite.

Reply:
Oh, right. I normally don't use this plugin so excuse my ignorance. I've tweaked the documentation a bit, which makes it a bit clearer I think.